### DISCUSSION TITLE
Is there a built in function that I can use for zoom in/out/rotate... I want to add a buttons to fire this events, so onClick eent on one button user will be able to zoom in for example

### DISCUSSION THREAD

**Discussion by RouaaAlzoubi** [google/model-viewer#2064] [Q&A]
### Description

Please provide a detailed description of the bug, how to reproduce it, and the
expected behavior.  Always include a code snippet, screenshot, any errors
reported in the console, and the model to help us understand and fix the
problem.

#### Live Demo

https://glitch.com/edit/#!/model-viewer

### Version

- model-viewer: vX.X.X

### Browser Affected

- [ ] Chrome
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [ ] Safari

### OS

- [ ] Android
- [ ] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

**Comment by elalish** [comment]
Yes, see `camera-orbit`, or it's JS version, `cameraOrbit`: https://modelviewer.dev/docs/index.html#entrydocs-stagingandcameras-attributes-cameraOrbit
